Assets/Controllers/GameManager.cs

```diff
@@ -153,4 +179,36 @@ public Vector3 GetPawnPosition(Pawn pawn)
         Vector3 pawnPosition = pawnKeeper.GetPawnGOFromPawn(pawn).transform.position;
         return pawnPosition;
     }
+
+    public void EndTurn()
+    {
+        int playerCount = players.Count;
```

Comment:
This seems a bit like a redundant declaration. You could easily use `players.Count` in your for loop and it is just as readable as playerCount.